README.md

```diff
@@ -763,6 +764,44 @@ Example output:
 }
 ```
 
+### Uploading Manifests
+
+ArDrive supports the creation of Arweave manifests using any of your PUBLIC folders or drives. This manifest will be created at the root of specified folder or drive just as any other ArFS file entity:
+
+```shell
+ardrive create-manifest -d bc9af866-6421-40f1-ac89-202bddb5c487 -w /path/to/wallet
+```
+
+The manifest data transaction is tagged with a unique content-type, `application/x.arweave-manifest+json`, which tells the gateway to treat this file as a manifest. The file itself is a .json file that holds the paths (the data transaction ids) to each file in the specified folder/drive.
```

Comment:
Some client education on gateways and manifests could be beneficial at the top of this section rather than at the bottom so that users can decide if it's something they want/need. This paragraph is the second mention of gateways anywhere in the doc, but the first mention only vaguely alludes to "indexing" without any explanation of their other roles and other capabilities. Consider adding something like:
```
Manifest files are special json files that instruct Arweave Gateways to map file data associated with specific, disparate transaction IDs to customized, hosted paths relative to that of the manifest file itself. So if, for example, your manifest file had an arweave.net URL like:

https://arweave.net/{manifest tx id}

then all the mapped txs and paths in the manifest file would be addressable at URLs like:

https://arweave.net/{manifest tx id}/foo.txt
https://arweave.net/{manifest tx id}/bar/baz.png
```
Then you could pretty seamlessly transition into how ArDrive produces and manages manifetss.